Documentation/ceph-nfs-crd.md

```diff
@@ -72,55 +82,395 @@ spec:
     priorityClassName:
 ```
 
- Enable the creation of NFS exports in the dashboard for a given cephfs or object gateway pool by running the following command in the toolbox container:
+## NFS Settings
+
+### RADOS Settings
+NFS configuration is stored in a Ceph pool so that it is highly available and protected. How that is
+configured changes depending on the Ceph version. Configuring the pool is done via the `rados` config.
+
+> **WARNING**: Do not use error corrected (EC) pools for NFS. NFS-Ganesha uses OMAP which is not
```

Comment:
Error corrected pools? Did you mean Erasure Coded pools? It's the first time I'm hearing this term in Ceph... If I missed something, how do people know they are using an error corrected pool?

Reply:
YES LOL! 

I'll add a link